constraint/pkg/externaldata/cache.go

```diff
@@ -69,6 +73,32 @@ func isValidURL(url string) bool {
 	return true
 }
 
+func isValidCABundle(provider *v1alpha1.Provider) error {
+	if provider.Spec.InsecureTLSSkipVerify {
+		if provider.Spec.CABundle != "" {
+			return fmt.Errorf("insecureTLSSkipVerify is set to true but caBundle is not empty")
```

Comment:
Is this necessarily bad? Might users be testing whether the CA bundle is bad in this scenario?

Reply:
Good point. I removed this condition to allow users to test their caBundle while InsecureTLSSkipVerify is enabled.